# Publications

## MIRPIPE: quantification of microRNAs in niche model organisms

Carsten Kuenne^1,‡^, Jens Preussner^1,‡^, Mario Herzog^1^, Thomas Braun^2^ and Mario Looso^1^

------------------    ---------------------------------------------------------------------------------------------------------------------
**Affiliations**      ^1^ Group of Bioinformatics and
                      ^2^ Department of Cardiac Development and Remodeling 
                          Max Planck Institute for Heart and Lung Research
                      ^‡^ These authors contributed equally
**Journal**           Bioinformatics, *Oxford Journals*
**Date, Issue**       2014, Vol. 30 no. 23
**Pages**             3412 - 3413
**DOI**               10.1093/bioinformatics/btu573
------------------    ---------------------------------------------------------------------------------------------------------------------


#### Contributions

The following contributions are attributed to the thesis autor:

| |    |
|-|---|
| **Conceptualization** | Contributed to definition of project goals |
| **Methodology** | Formulation of the graph-based isoMiR handling; Formulation of the benchmark tests |
| **Investigation** | Contributed to interpretation and discussion of results |
| **Validation** | Benchmarking reproducibility and predictive efficiency |
| **Software** | Contributed to overall workflow implementation; Implemented the graph-based isoMiR handling |
| **Visualization** | Created Figure 1 and Supplementary File S2 |
| **Resources ** | Setup and administration of MIRPIPEs web service; Contributed to software manual |
| **Writing** | Contributed to manuscript draft; Review and editing of the manuscript |

```{r, results='asis', echo=FALSE, eval=knitr::is_latex_output()}
cat("\\includepdf[pages={-}]{publications/2014_mirpipe.pdf}")
```

## ADMIRE: analysis and visualization of differential methylation in genomic regions using the Infinium HumanMethylation450 Assay

Jens Preussner^1^, Julia Bayer^1^, Carsten Kuenne^1^ and Mario Looso^1^

------------------    ---------------------------------------------------------------------------------------------------------------------
**Affiliations**      ^1^ Group of Bioinformatics
                          Max Planck Institute for Heart and Lung Research
**Journal**           Epigenetics and Chromatin, *BioMed Central*
**Date, Issue**       2015, Vol. 8 no. 1
**Pages**             1 - 10
**DOI**               10.1186/s13072-015-0045-1
------------------    ---------------------------------------------------------------------------------------------------------------------

#### Contributions

The following contributions are attributed to the thesis autor:

| |    |
|-|---|
| **Conceptualization** | Overall definition of project goals; Definition of visualizations |
| **Methodology** | Design of the software; Formulation of ADMIREs differential testing procedure; Formulation of the benchmark tests |
| **Investigation** | Conducted analysis of the RA and AF datasets; Contributed to interpretation and discussion of results |
| **Validation** | Benchmarking ADMIRE against RnBeads; Evaluation of ADMIREs performance |
| **Software** | Implementation of the ADMIRE workflow |
| **Visualization** | Creation of Fig. 1 to Fig. 4, Additional files 4 to 8 |
| **Resources ** | Setup and administration of ADMIREs web service; Wrote the manual |
| **Writing** | Contributed to the manuscript draft; Review and editing of the manuscript |

```{r, results='asis', echo=FALSE, eval=knitr::is_latex_output()}
cat("\\includepdf[pages={-}]{publications/2015_admire.pdf}")
```

## A molecular subtype of cancer originating from adult stem cells during regeneration is driven by Dux transcription factors

Jens Preussner^1,2,‡^, Jiasheng Zhong^1,‡^, Krishnamoorthy Sreenivasan^1^, Stefan Günther^1,3^, Thomas Engleitner^4^, Carsten Künne^1,2^, Markus Glatzel^5^, Roland Rad^4^, Mario Looso^2^, Thomas Braun^1,6,7^ and Johnny Kim^1,6^*

------------------    ---------------------------------------------------------------------------------------------------------------------
**Affiliations**      ^1^ Department of Cardiac Development and Remodeling
                      ^2^ Bioinformatics Core Unit (BCU)
                      ^3^ ECCPS Deep Sequencing Platform
                          Max Planck Institute for Heart and Lung Research
                      ^4^ Institute of Molecular Oncology and Functional Genomics
                          Translatum Cancer Center and Department of Medicine II
                          Technical University of Munich
                      ^5^ Institute of Neuropathology
                          University Medical Center Hamburg-Eppendorf
                      ^6^ German Center for Cardiovascular Research (DZHK)
                      ^7^ German Center for Lung Research (DZL)
                      ^‡^ These authors contributed equally
**Journal**           Cell Stem Cell, *Cell Press*
**Date, Issue**       2018, Vol. 23 no. 6
**Pages**             794 - 805
**DOI**               10.1016/j.stem.2018.10.011
------------------    ---------------------------------------------------------------------------------------------------------------------

#### Contributions

The following contributions are attributed to the thesis autor:

| |    |
|-|---|
| **Conceptualization** | Contributed to definition of project goals; Outlined data analysis strategy |
| **Methodology** | Hypothesis formulation of the role of Duxbl in epithelialization |
| **Investigation** | Analysis of recurring SNVs and CNVs, previously published discovery cohorts, data from TCGA; Investigation of the role of Dux family genes in ZGA; Analysis of previously published data from RNA-seq to investigate the role of Duxbl in epithelialization |
| **Validation** | *no contribution* |
| **Software** | Implementation of software to analyse data from whole-exome sequencing |
| **Visualization ** | Figures 3C, 3F, 3G, Figure 4, Figure 6 and graphical abstract; Supplementary Figures S3 to S7 |
| **Resources ** | Tables S1 and S2 supporting CN and SNV analysis; Tables S3, S4 supporting findings from TCGA analysis | 
| **Writing** | Contributed to the manuscript draft; Review and editing of the manuscript |

```{r, results='asis', echo=FALSE, eval=knitr::is_latex_output()}
cat("\\includepdf[pages={-}]{publications/2018_duxbl.pdf}")
```
