This thesis comprises three peer-reviewed publications, which are presented in chronological order.

#### Publication 1 {-}

Carsten Kuenne, Jens Preussner, Mario Herzog, Thomas Braun and Mario Looso. **MIRPIPE: quantification of microRNAs in niche model organisms**. *Bioinformatics*, Vol. 30 (2014)

The publication introduces MIRPIPE, an algorithm for rapid miRNA detection and quantification from NGS data. The algorithm takes raw data from *total RNA sequencing* as input and initially performs quality control by filtering out reads with low quality base calls. Reads that are too long or too short and do not match the length assumption of mature miRNAs (18 - 28*nt*) are eliminated, and sequencing adapter contamination from the *3'*-end is eradicated. MIRPIPE optionally removes unique, or low abundant reads frequently denoting remaining sequencing errors or miRNA variations below the detection limit and clusters reads sharing the same *5'*-end to properly handle miRNAs originating from the same gene but imperfectly digested by Drosha/Dicer (isoMiRs). MIRPIPE builds a graph from homology searches against a reference database with reference miRNAs as nodes and edges when read(s) support two reference miRNAs equally well. This unique approach permits inclusion of reads that otherwise cannot be matched uniquely and detects miRNA families as connected components in the graph. Quantification results, *i.e.* counts per miRNA family and miRNA cluster (isoMiRs) can readily be used for downstream differential expression analysis.

#### Publication 2 {-}

Jens Preussner, Julia Bayer, Carsten Kuenne and Mario Looso. **ADMIRE: analysis and visualization of differential methylation in genomic regions using the Infinium HumanMethylation450 Assay**. *Epigenetics and Chromatin*, Vol. 8 (2015)

The publication introduces ADMIRE, a pipeline for differential methylation analysis within genomic regions. The algorithm takes raw data from *Infinium HumanMethylation450 BeadChips* and initially filters single CpG probes based on low signal-to-noise ratios in a variable proportion of analysed samples. ADMIRE offers several techniques to perform between-sample normalization, *e.g.* by regressing out variability observed between control probes, before calculating two one-sided two-sample rank tests per CpG probe and between any two sample groups. Intentionally, two p-values are obtained per CpG probe, indicating lower or higher methylation in either group. ADMIRE allows subsequent combination of spatially correlated p-values into arbitrary genomic regions using weighted Z-scores. It controls the familywise error rate as well to accomplish multiple testing correction. Differential methylation results are used to perform gene set enrichment analysis, when genomic regions can be meaningfully related to genes, thereby facilitating target selection in clinical settings. Further, most significantly altered regions are readily visualised and exported for use in genome browsers or in spreadsheet viewers.

#### Publication 3 {-}

Jens Preussner, Jiasheng Zhong, Krishnamoorthy Sreenivasan, Stefan Günther, Thomas Engleitner, Carsten Künne, Markus Glatzel, Roland Rad, Mario Looso, Thomas Braun and Johnny Kim. **A molecular subtype of cancer originating from adult stem cells during regeneration is driven by Dux transcription factors**. *Cell Stem Cell*, Vol. 23 (2018)

The publication investigates the cellular and molecular origin of rhabdomyosarcoma leveraging lineage tracing, genomic and transcriptomic analysis. Using the mdx mouse model, we show that genetic inactivation of the Tp53 tumour suppressor in Pax7-expressing muscle stem cells located in continuously regenerating skeletal muscles will give rise to fusion-negative RMS. The approach allowed tracing of the cellular origin of tumours back to transformed muscle stem cells. Genomic analysis of purified tumour-propagating cells reveals large genomic instability and identifies diverse genomic lesions, among them known oncogenes and cancer-promoting pathways. An amplicon was identified that contained a member of the Dux family of transcription factors, pointing to a novel candidate oncogene in RMS. Expression analysis of several published RMS studies and extended analysis of data from the Cancer Genome Atlas suggests that Dux-expressing tumours represent a distinct subtype of fusion-negative RMS and a novel pan-cancer subtype. Analysis of data from forced expression of Duxbl in muscle stem cells revealed that Duxbl can elicit epithelialization/colonization via a MET-like program to initiate tumour formation. Lastly, we investigate therapeutic intervention of Duxbl tumours via short hairpin mediated knockdown. 
