### Epigenetic mechanisms in cancer initiation and progression {#epigenetic}

Development of normal tissue, as discussed in chapter \@ref(regeneration), requires distinct cell types to arise during lineage-specification. Although equipped with identical genomic information, different cell types exhibit different gene expression programs and are able to pass such information on to their progeny. How are such expression patterns specified and maintained? Nowadays, it is accepted that not only information encoded as DNA in a cells genome, but also *epigenetic* information (i.e. the stable and heritable non-genetic counterpart to DNA) provides an important layer of regulation and plays pivotal roles in cell lineage specification and cell identity maintenance [@margueronChromatinStructureInheritance2010]. In eukaryotes, DNA is organized in a massive macromolecular complex called chromatin. Chromatin is formed by wrapping 147 base pairs of DNA around a histone octamer (nucleosomes), then compacting those further into topologically associated domains (TADs) separated by insulator proteins to allow independent and specific regulation. The activity of a genomic locus is controlled by its chromatin organisation. Accessible chromatin structures expose DNA elements, like proximal gene promoter sequences or distal enhancer sequences, to regulatory transcription factors and the transcriptional machinery to drive gene expression. Compact and inaccessible chromatin structures prevent such activity and render a locus inactive. 

Dynamic changes needed during tissue development call for mechanisms able to alter chromatin organisation in response to changed conditions [@johnDevelopmentalEpigeneticsPhenotype2018]. Chromatin remodelling resulting in transcriptional repression is, for example, enforced by the Polycomb protein family, which can post-transcriptionally modify specific histone residues (e.g. trimethylation of histone H3, lysine 27 (H3K27me3)). Repressive chromatin states can be propagated through cell division by retention of catalytic enzymes on replicating DNA [@simonOccupyingChromatinPolycomb2013] and functional interaction with DNA methylation and other regulatory proteins [@flavahanEpigeneticPlasticityHallmarks2017]. Conversely, regulatory activity by e.g. binding of transcription factors and chromatin modifiers, seems to block repressive chromatin compaction [@zaretPioneerTranscriptionFactors2016]. Further, active loci that are marked by trimethylation of histone H3, lysine 4 (H3K4me3), in turn inhibit recruitment of DNA methyltransferases for *de novo* DNA methylation [@ooiDNMT3LConnectsUnmethylated2007], which is another potent epigenetic mechanism for stabilization of transcriptional repression [@jonesFunctionsDNAMethylation2012]. Methylated DNA functions as *silencing* mark and is involved in processes like X-chromosome inactivation [@venoliaComparisonTransformationEfficiency1983], repression of transposable and repetitive DNA elements [@yoderCytosineMethylationEcology1997] and might influence genome function when present at regulatory elements, like enhancers or chromatin insulators [@bellMethylationCTCFdependentBoundary2000]. 

A compelling conceptualization of epigenetic regulation during cell lineage development has been postulated by developmental biologist Conrad H. Waddington, outlined in his assay entitled *The strategy of genes* more than 60 years ago (reprinted in @waddingtonStrategyGenes2014). In his hypothesis, differentiating cells proceed downhill along branching valleys in an energetic landscape (Fig. \@ref(fig:epigenetic-landscape)a). The valleys correspond to discrete cellular states and their topological layout is defined by underlying gene regulatory networks (GRN) that actively shape and maintain cellular identity [@zaretPioneerTranscriptionFactors2016]. Walls between valleys restrict cell lineage capacity, by preventing cells to randomly *switch states* (i.e. hopping over to another valley), and epigenetic mechanisms effectively modulate the height of walls. Compacted and repressing chromatin, for example, prevents spurious activation of non-lineage gene regulatory factors, restricting changes in gene activity and increasing the height of energy walls between cell states, which blocks changes in cell state and cell type identity. 

```{r epigenetic-landscape, fig.scap = "The epigenetic landscape in lineage development and cancer initiation.", fig.cap="The epigenetic landscape in lineage development and cancer initiation. (a) Depiction of normal lineage development in the conceptualized epigenetic landscape from C. Waddington. Stem cells (blue) reside at the apex of a lineage and exhibit high lineage capacity. Progenitor cells proceed downhill along branching valleys into differentiated cell types. Oncogenetic events (black lightning) in lineage progeny with high lineage capacity can create cancer stem cells by switching cell states and might lead to development of cancer (red arrow). (b) Restrictive chromatin might arrest progenitor cells in their proliferative state, prevent their terminal differentiation and lead to cancer initiation. (c) Deteriorated and permissive chromatin confers enhanced cellular plasticity and might lead to spurious gene activation or cellular reprogramming and predisposes for cancer initation.", fig.align="center", dpi=300, out.width="146.5mm", message=FALSE, warning=FALSE, error=FALSE}

knitr::include_graphics("figures/epigentic_landscape-01.pdf")
```

Initiation and progression of cancer can result from various mechanisms disrupting normal epigenetic regulation. Overly restrictive chromatin (i.e. high energy walls between valleys of Waddingtons landscape, Fig. \@ref(fig:epigenetic-landscape)b) can be achieved by gain-of-function mutations of Ezh2, the catalytic subunit of Polycomb repressive complex 2 (Prc2). A hyperactive methyltransferase activity of Ezh2 leads to expansive, genome-wide H3K27 methylation [@sneeringerCoordinatedActivitiesWildtype2010] and the loss of active chromatin marks. Overly restrictive chromatin arrests developing B-cells in a proliferative state and prevents terminal differentiation, leading to B-cell lymphoma [@beguelinEZH2RequiredGerminal2013]. Epigenetic restriction can also arise from aberrations in DNA methylation: The CpG island methylator phenotype (CIMP) results from DNA hypermethylation and is characterized by silencing of tumour suppressor genes and DNA mismatch repair genes [@hitchinsInheritanceCancerAssociatedMLH12007]. Deterioration of overall chromatin topology (the layout of Waddingtons landscape, Fig. \@ref(fig:epigenetic-landscape)c) can be achieved by disruption of CTCF binding [@flavahanInsulatorDysfunctionOncogene2016], a methylation-sensitive DNA binding protein that accomplishes partitioning of chromosomal loops into functional units by insulating TADs. Regulatory TAD boundaries protect against gene activation from overly promiscuous enhancers from neighboring TADs and loss thereof can lead to the activation of oncogenes [@hniszActivationProtooncogenesDisruption2016]. Upregulation of members of the Histone Lysine Demethylase (Kdm) protein family has been implicated in formation of overly permissive chromatin (i.e. low energy walls between valleys). Enhanced epigenetic plasticity allows for rapid cell reprogramming or adaptation and drives diverse cancer types [@roeschOvercomingIntrinsicMultidrug2013; @blackHypoxiaDrivesTransient2015; @liauAdaptiveChromatinRemodeling2017]. Finally, loss of imprinting through DNA hypomethylation may permit reactivation of oncogenes, e.g. the insulin growth factor signalling pathway in some sarcomas [@rikhofInsulinlikeGrowthFactor2009]. In such cases, both, the maternal and paternal copy of the Igf2 gene are transcribed, leading to elevated mRNA levels and predispose for cancer through autocrine signalling.
