### Epigenetic mechanisms in cancer initiation and progression {#epigenetic}

Development of normal tissue, as discussed in chapter \@ref(regeneration), requires distinct cell types to arise during lineage-specification. Although of identical genomic information, different cell types exhibit different gene expression programs and are able to pass such information on to their progeny. How are such expression pattern specified and maintained? Nowadays, it is accepted that not only information encoded as DNA in a cells genome, but also *epigenetic* information (i.e. the stable and heritable non-genetic counterpart to DNA) provides an important layer of regulation and plays pivotal roles in cell lineage specification and cell identity maintenance [@margueron_chromatin_2010]. In eurkaryotes, DNA is organized in a massive macromolecular complex called chromatin. Chromatin is formed by wrapping 147 base pairs of DNA around histone proteins into nucleosomes, then compacting those further into topologically associated domains (TADs) separated by insulator proteins to allow independent and specific regulation. The activity of a genomic locus is controlled by its chromatin organization. Accessible chromatin structures expose DNA elements, like proximal gene promoter sequences or distal enhancer sequences, to regulatory transcription factors and the transcriptional machinery to drive gene expression. Compact and inaccessible chromatin structures prevent such activity and render a locus inactive. 

Dynamic changes needed during tissue development call for mechanisms to alter chromatin in response to changed conditions or appropriate cues, e.g. cell-to-cell signalling (Citation). Chromatin remodelling resulting in transcriptional repression is, for example, enforced by the Polycomb protein family, which can post-transcriptionally modify specific histone residues (e.g. methylation of histone H3, lysine 27 (H3K27me3)). Repressive chromatin states can be propagated through cell division by retention of catalytic enzymes on replicating DNA [@simon_occupying_2013] and functional interaction with DNA methylation and other regulatory proteins [@flavahan_epigenetic_2017]. Conversely, regulatory activity by e.g. binding of transcription factors and chromatin modifiers, seems to block repressive chromatin compaction [@zaret_pioneer_2016]. Further, active loci that are marked by trimethlyation of histone H3, lysine 4 (H3K4me3), in turn inhibit recruitment of DNA methyltransferases for *de novo* DNA methylation [@ooi_dnmt3l_2007], which is another potent epigenetic mechanism for stabilization of transcriptional repression [@jones_functions_2012]. Methylated DNA functions as *silencing* mark and is, for example, involved in X-chromosome inactivation [@venolia_comparison_1983], repressesion of transposable and repetetive DNA elements [@yoder_cytosine_1997] and might influence genome function when present at regulatory elements, like enhancers or chromatin insulators [@bell_methylation_2000]. 

An influential conceptualization of epigenetic regulation during cell lineage development has been brought up by developmental biologist Conrad H. Waddington, outlined in his assay entitled *The strategy of genes* more than 60 years ago (reprinted in @waddington_strategy_2014). In his understanding, differentiating cells proceed downhill along branching valleys in an energetic landscape. The valleys correspond to discrete cellular states and their topological layout is defined by underlying gene regulatory networks (GRN) that actively shape and maintain cellular identity [@zaret_pioneer_2016]. Walls between valleys restrict cell lineage capacity, by preventing cells to randomly *switch states* (i.e. hopping over to another valley), and epigenetic mechanisms effectively modulate the height of walls. Compacted and repressing chromatin, for example, prevents spurious activation of non-lineage gene regulatory factors, restricting changes in gene activity and increasing the height of energy walls between cell states, which blocks changes in cell state and cell type identity. 

Initiation and progression of cancer can result from multiple modes of disruption of epigenetic regulation, ranging from overly restrictive chromatin (i.e. high energy walls between valleys of Waddingtons landscape), disruption of chromatin topology (i.e. loss of regulatory TAD boudaries) to overly permissive chromatin (i.e. low energy walls between valleys). Gain-of-function mutations of EZH2, the catalytic subunit of Polycomb repressive complex 2 (PRC2), results in a hyperactive methlytransferase activity and expansive, genome-wide H3K27 methylation [@sneeringer_coordinated_2010], accompanied by a loss of active chromatin marks. The overly restrictive chromatin arrests developing B-cells in a proliferative state and prevents terminal differentiation, leading to B-cell lymphoma [@beguelin_ezh2_2013]. Epigenetic restriction can also arise from aberrations in DNA methylation: The CpG island methylator phenotype (CIMP) results from DNA hypermethylation and is characterized by silencing of tumor supressor genes and DNA mismatch repair genes [@hitchins_inheritance_2007]. DNA hypermethlyation can also disrupt binding of methylation-sensitive CTCF [@flavahan_insulator_2016], an important DNA binding protein that accomplishes partitioning of chromosomal loops into functional units by insulating TADs and thereby protecting gene activation from overly promiscuous enhancers from neighboring TADs [@hnisz_activation_2016]. Upregulation of members of the Histone Lysine Demethylase (KDM) protein family has been implicated in formation of diverse cancer by enhancing epigenetic plasticity, allowing for rapid cell reprogramming or adaptation [@roesch_overcoming_2013; @black_hypoxia_2015; @liau_adaptive_2017]. Finally, loss of imprinting (DNA hypomethylation) may permit reactivation of oncogenes, e.g. the insulin growth factor signalling pathway in some sarcomas [@rikhof_insulin-like_2009]. In such cases, both, the maternal and paternal copy of the IGF2 gene are transcribed, leading to elevated mRNA levels and predispose for cancer through autocrine signalling. 
