### Small RNA mediated carcinogenesis {#mirna}

Micro RNAs (miRNAs) are small, non-coding RNAs of ~22 nucleotides and serve numerous roles in negative gene regulation. In animals, most miRNAs exhibit their regulatory role through imperfect binding of a sequence in the 3' untranslated region (3'-UTR) of messenger RNA from target genes. Complementary binding can either repress translation of target gene(s) or mediate mRNA degradation, through a mechanism similar to RNA interference in plants [@jones-rhoadesMicroRNAsTHEIRREGULATORY2006].  

MiRNA biogenesis begins with the transcription of either independent miRNA genes or intronic regions from protein-coding genes into large precursor molecules (pri-miRNAs). Imperfect base-paring of folding pri-miRNAs results in hairpin structures that are further cleaved by an RNase III type endonuclease (Drosha, RN3) together with a double-stranded RNA binding domain (dsRBD) protein [@hanDroshaDGCR8ComplexPrimary2004] into ~70 nucleotide hairpins called pre-miRNAs, leaving a short characteristic single-stranded overhang at the 3'-end of pre-miRNAs. Exportin 5 recognizes such an overhang and arranges the transport of pre-miRNAs to the cytoplasm [@yiExportin5MediatesNuclear2003], where a second complex consisting of Dicer, a RNase III type enzyme, and TRBP, a dsRBD protein, cleave the pre-miRNA twice into a miRNA duplex [@chendrimadaTRBPRecruitsDicer2005]. One strand (the mature miRNA) preferentially enters the miRNA-induced silencing complex (miRISC), while the other is degraded, although the complementary miRNA is also competent for miRNA-mediated silencing [@schwarzAsymmetryAssemblyRNAi2003]. Imperfect double-strand pairing of pre-miRNAs as well as imperfect digestion by Drosha and Dicer result in miRNAs with varying 3'- (silent modification, *isomiR*) or 5'-ends, that might affect complementary binding, representing a challenge for computational miRNA quantification following RNASeq. 

Forward genetic experiments have revealed great importance of the miRNA machinery, exemplified by muscle-specific knockout of Dicer leading to complete embryonic development but perinatal death [@bernsteinDicerEssentialMouse2003]. Other experiments identified numerous individual miRNAs with roles in processes, like timing of development [@abrahanteCaenorhabditisElegansHunchbacklike2003], differentiation [@chenMicroRNAsModulateHematopoietic2004] and growth control [@brenneckeBantamEncodesDevelopmentally2003]. Since phenotypic consequence remains elusive for the vast majority of miRNA, also computational approaches can and have been used to elucidate miRNA function [@ulitskyComputationalPredictionMicroRNA2010; @liuPredictionFunctionalMicroRNA2019]. 

Progression of cancer growth can also be altered by expression of certain miRNAs. The dedifferentiated phenotype of ERMS, for example, can result from downregulation of muscle-specific miRNAs (*myomiRs*), that promote myogenic differentiation under normal conditions. Expression of miR-1/206, a skeletal and cardiac muscle-specific miRNA, induces cell differentiation in C2C12 myoblast cells [@kimMusclespecificMicroRNAMiR2062006] by downregulation of Pax3 and inhibition of cell migration. Additionally, targeting of Pax7 by miR-206/486 stops proliferation and activates MyoD, allowing for muscle differentiation [@chenMicroRNA1MicroRNA206Regulate2010]. Transfection of miR-1/206 into a RMS cell line notably decreased tumour cell migration and proliferation even more than switching to a differentiation medium [@taulliMusclespecificMicroRNAMiR2062009]. Further, miR-1/206 was shown to target c-Met signalling, leading to a decrease in tumour mass, while downregulation of miR-1/206 resulted in c-Met upregulation and RMS progression [@yanMicroRNA1206Targets2009]. Non-myomiRs can as well promote myogenic differentiation, *e.g.* miR-26a mediates downregulation of cell-cycle progression by targeting Ezh2. Vice versa, downregulation of miR-26a in RMS results in upregulation of Ezh2 and therefore prevents myogenic differentiation [@ciarapicaDeregulatedExpressionMiR26a2009]. Amplification of 13q31 in 25% of ARMS cases results in enhanced expression of the miR-17-92 cluster (*oncomiR-1*), a *bona fide* oncogene [@reichekGenomicClinicalAnalysis2011; @sandhuBcellMalignanciesMicroRNA2013; @jinMicroRNA1792Plays2013a], potentially targeting tumour suppressors like PTEN. Deregulation of another oncogene in RMS, miR-183, is reported to promote tumour cell migration, by targeting the transcription factor Egr1, a direct regulator upstream of other tumour suppressor genes [@sarverMicroRNAMiR183Functions2010a; @mohamadEGR1InteractsTBX22018].

