### Knowledge gaps

The questions of how and why Rhabdomyosarcoma develops has already led to several studies set out to characterize the cellular origin and molecular alterations in RMS. In an extensive genomic analysis of tumours from 13 patients, Chen et. al.[@chenTargetingOxidativeStress2013] report an increased background mutation rate, increased number of SVs and CNVs in tumours from ERMS patients, compared to tumours from ARMS patients. However, no recurrent alteration was common to more than two patients and no hints for chromotripsis, as an explanation for large numbers of SVs, were observed. Instead, they report large genomic heterogeneity at sites of known oncogenic drivers, like in Ras family genes or SHH/Wnt signalling. The somatic mutation theory introduced earlier (see chapter \@ref(smt)) puts forward the need for progressive accumulation of genome alterations that allows the cell to escape control. Together with observations from Chen et. al., two open questions can be answered: Which genomic mutations occur in RMS and are mutations reoccurring? RMS, and especially the ERMS subtype accumulated both, single nucleotide polymorphisms and larger copy number variations. Genomic alterations were not reoccurring among cancer samples and provide a large cancer heterogeneity - a result that follows observations from large genome sequencing projects, although in case of Chen et. al, only 16 samples were probed. Extending the somatic mutation theory, the genome theory explains cancer heterogeneity by introducing karyotype heterogeneity. Further, the genome theory assumes that initial genome context instability ignites a series of genome alterations from which *fittest* (in terms of cell proliferation and expansion) can be selected by an evolutionary process. An open question thus is which mechanisms induce genomic instability in RMS and how do accumulated genome alterations confer to fitness? Which hallmarks do they operate?

Following the stem cell theory of cancer (see chapter \@ref(csc)), the cancer cell of origin of embryonic rhabdomyosarcoma could possibly be either the muscle stem cell itself, or any of its potent progenitor cells, e.g. activated stem cells or pro-myogenic precursor cells. Elevated expression of Pax7 exclusively in fusion-negative RMS led (@tiffinPAX7ExpressionEmbryonal2003) to propose their origin within the myogenic lineage. Transformation of precursor cell populations from the myogenic lineage, with either expression of oncogenic Kras [@hettmerSarcomasInducedDiscrete2011; @blumDistinctOverlappingSarcoma2013], or lineage-specific deletion of tumour suppressor genes [@rubinEvidenceUnanticipatedRelationship2011], led to tumours that correlate with their *presumable* myogenic origin. However, it was not possible to discriminate the cellular origin of Kras-expressing tumours with transcriptional analysis thereof [@hettmerSarcomasInducedDiscrete2011]. The studies presented above lack direct experimental evidence, by leaving the possibility that other cell types could act as tumour initiator in the muscle compartment by e.g. cell migration mechanisms. Therefore, the cellular origin of RMS ultimately remains to be disclosed.

MiRNAs as well play a role in cancer initiation, progression and maintenance (see chapter \@ref(mirna)) and discovery of the entire regulatory repertoire of these small molecules is crucial for understanding their function in a given biological system. With the application of next generation sequencing in miRNA research [@tamRobustGlobalMicroRNA2014], numbers of identified miRNAs increased rapidly, as well as computational approaches to predict or detect them [@gomesReviewComputationalTools2013]. However and until recently, *isomiR* variation was dismissed as sequencing artefact and led to underestimation of the *miRNome* complexity [@neilsenIsomiRsOverlookedRepertoire2012], which constitute a challenge for proper computational detection and quantification of miRNAs. Further, miRNA detection often relies on the presence of a preferably complete genomic reference to align miRNA reads to a genomic locus or reference database, or extensive homology searches to exploit evolutionary conservation of a nearby species. Open questions in computational analyses of data from miRNA-seq thus include *isomiR* handling and whether or not a reference-free approach to miRNA detection is as sensitive as the conventional approach and can be used to also incorporate knowledge from other organisms. 
To functionally characterize miRNAs, the delineation of miRNA target genes, i.e. those genes that could be silence by complimentary miRNA binding, is required and often accomplished using *in silico* target prediction tools (reviewed in (@oulasPredictionMiRNATargets2015)). Although prior knowledge from databases harbouring validated miRNA target interactions (MTIs) exists, target prediction and integration of other experimental data, e.g. from expression studies, remains challenging.

Comparably, epigenetic regulation plays an important role in cell lineage development and disruption of such regulation can serve tumour initiation as well as tumour maintenance (see chapter \@ref(epigenetic)). Mechanistic insight into how epigenetic lesions take effect (also in cooperation with or followed by ordinary genetic stimuli) is however missing. Open questions with diagnostic and therapeutic implications include whether or not an initiating genetic hit (e.g. gain or loss of function) becomes secondary, once a downstream epigenetic lesion has occurred and altered the cellular state towards permanent tumourigenicity. Advances in microarray and next generation sequencing technology make it possible to assay different mechanisms of epigenetic regulation at high-resolution and in a large number of samples [@theencodeprojectconsortiumIdentificationAnalysisFunctional2007; @listerFindingFifthBase2009] but require specialized computational analysis (reviewed in @bockComputationalEpigenetics2008). Subsequently, discussion is focused on DNA methylation as mechanism of epigenetic regulation, since it is intensely studied and it's widespread implication in cancer is well understood [@kulisDNAMethylationCancer2010; @sekiIntegratedGeneticEpigenetic2015; @sunRelationshipDNAMethylation2019]. Profiling DNA methylation using microarray technology [@bibikovaHighDensityDNA2011] allows researchers to assay large number of samples across the whole genome. Computational analysis of data from such technologies aims to identify differentially methylated regions between two or more groups and search for functional enrichment in those regions [@lairdPrinciplesChallengesGenomewide2010; @bockAnalysingInterpretingDNA2012]. However, such approaches are complicated by the spatial interdependencies of individual CpG sites. Open questions include how data from single CpG sites can be aggregated into regions in order to capture higher-order methylation patterns across broader genomic regions and if such data can be analysed without the need of prior knowledge, e.g. the definition of genomic regions, location of CpG islands, promoters or other gene regulatory regions. For further downstream interpretation of results, annotation of differentially methylated regions to nearby genes or known genomic features is not straightforward and requires flexible and fast software solutions.

### Objectives

Based on the previously described shortcomings of currently available methods for analysis of data obtained from high throughput experiments, two of the three main objectives of this thesis focus on methodological improvements in miRNA quantification and the analysis of DNA methylation pattern in arbitrary genomic regions. Another objective aims at the disclosure of cellular and molecular origins of Rhabdomyosarcoma, using multilayer datasets, applied bioinformatics and advanced methods from cell biology. Ultimately, methodological advancements will help to interpret and understand data from high-throughput experiments in Rhabdomyosarcoma and potentially lead to the disclosure of molecular pathways acting during tumour initiation and progression.

#### Reference-free and fine-grained analysis of data from miRNA-sequencing

The large number of miRNAs detected from previous high-throughput experiments across many species and experimental conditions represents a great data resource to guide detection and quantification of newly-created datasets. Sensitive homology searches versus those existing miRNA sequences could be a promising starting point for genome reference-free miRNA detection. A graph-based data structure with known miRNA sequences as nodes, and edges representing sequencing read(s) with a match to known miRNA sequences equally well, could enable further fine-grained analyses: We postulate that closely connected components of the graph represent miRNA family's across different *isomiR*s and species, and allows for improved quantification on either family or *isomiR* levels by modulation of sensitivity of the homology search. Further, the sum of edge weights could readily be fed into differential expression analysis, enabling streamlined downstream analysis.

#### Analysis of DNA methylation pattern in arbitrary genomic regions

Although a plethora of methods exist for data normalization and single-site CpG methylation analysis, a robust method that takes the spatial interdependencies of CpG sites into account and allows for unbiased genome-wide analysis is critically missing. Combining differences of CpG site methylation estimated via one-sided two-sample Wilcoxon rank tests with a recently published method for grouping and correction of spatially correlated p-values [@pedersenCombpSoftwareCombining2012] could represent a novel and powerful statistical approach to detect differentially methylated regions. Importantly, this approach would eliminate the need for *a priori* knowledge of regulatory regions and allow for arbitrary genomic regions to be evaluated. Integration of numerous existing normalization techniques and automated downstream gene set enrichment methods could additionally enhance usability and open the method for a wide variety of applications.

#### The cellular and molecular origin of Rhabdomyosarcoma

Existing studies set out to delineate the cellular and molecular origin of Rhabdomyosarcoma did not provide direct evidence for involvement of the myogenic lineage in tumour formation, but rather based their (valuable) findings on correlation with myogenic properties. The Cre/lox site-specific recombination system has been developed to create time- and tissue-specific mutations, e.g. to study effects of inducible gene knockouts [@feilInducibleCreMice2009]. We reasoned that employment of an inducible recombination system could provide permanent fluorescence labelling of muscle stem cell as well as their progeny and therefore provide direct evidence of muscle stem cell tumourigenicity. Further, it would enable subsequent genomic and epigenomic analysis of purified tumour-propagating cells, as well as investigation of clonal evolution in secondary recipients.

