The question of how and why cancer develops already has led to numerous studies to characterise molecular tumour profiles and the cellular origin of different cancer types. Methods from molecular biology and high-throughput assays (e.g. array or sequencing-based) are often used in conjunction to draw conclusions and validate results. A plethora of computational methods have been implemented for analysis of data from such assays and form the basis for approaches integrating across different data types. The upcoming sections will line out major knowledge gaps in computational approaches to handle data from high-throughput assays and in the search of cancer origin and cancer subtype detection, exemplified using ERMS. Further, objectives of this thesis were developed from identified knowledge gaps.

### Knowledge gaps

In an extensive genomic analysis of tumours from 9 ERMS patients, @chenTargetingOxidativeStress2013 set out to define biological signatures to predict patient outcome and assign targeted therapy for a high-risk subtype. They identified a subtype with defects in oxidative metabolism, but otherwise report a multiplicity of mutations in known cancer consensus pathways such as in Ras family genes, SHH/Wnt signalling or cell-cycle checkpoints. Analysis of therapy-resistant tumour subclones highlighted complex genetic changes and clonal evolution. The observations from Chen et. al., can answer two open questions: Which genomic mutations occur in ERMS and are mutations reoccurring? The ERMS subtype accumulated both, single nucleotide polymorphisms and larger copy number variations. Genomic alterations were not reoccurring among cancer samples and indicate a large tumour heterogeneity - a result that complicates further subtype detection and follows observations from large genome sequencing projects. Extending the somatic mutation theory (see chapter \@ref(smt)), the genome theory explains cancer heterogeneity by introducing karyotype heterogeneity. Further, the genome theory assumes that initial genome context instability ignites a series of genome alterations from which the *fittest* (in terms of cell proliferation and expansion) is selected by an evolutionary process. An open question thus is which mechanisms induce genomic instability in ERMS and how do accumulated genome alterations confer to fitness? Which hallmarks do they operate and do these specify further ERMS subtypes?

Following the stem cell theory of cancer (see chapter \@ref(csc)), the cancer cell of origin of ERMS could possibly be either the muscle stem cell itself, or any of its potent progenitor cells, e.g. activated stem cells or pro-myogenic precursor cells. Elevated expression of Pax7, which occurs exclusively in fusion-negative RMS, led @tiffinPAX7ExpressionEmbryonal2003 to propose the origin of RMS within the myogenic lineage. Transformation of precursor cell populations from the myogenic lineage with either expression of oncogenic Kras [@hettmerSarcomasInducedDiscrete2011; @blumDistinctOverlappingSarcoma2013], or lineage-specific deletion of tumour suppressor genes [@rubinEvidenceUnanticipatedRelationship2011], led to tumours that phenotypically resembled their *presumable* myogenic origin. However, it was not possible to discriminate the cellular origin of Kras-expressing tumours by transcriptional analysis [@hettmerSarcomasInducedDiscrete2011]. The studies presented above lack direct experimental evidence, leaving the possibility that other cell types in the muscle compartment might act as tumour initiator by e.g. cell migration mechanisms. Therefore, the cellular origin of RMS ultimately remained to be disclosed.

MiRNAs play a role in cancer initiation, progression and maintenance (see chapter \@ref(mirna)) and discovery of the entire regulatory repertoire of these small molecules is crucial for understanding their function in a given biological system. With the application of next generation sequencing in miRNA research [@tamRobustGlobalMicroRNA2014], the numbers of identified miRNAs increased rapidly, as well as computational approaches to predict or detect them [@gomesReviewComputationalTools2013]. However, until recently, *isomiR* variation due to imperfect digestion of pre-miRs by Drosha and Dicer was dismissed as sequencing artefacts and led to underestimation of the *miRNome* complexity [@neilsenIsomiRsOverlookedRepertoire2012], which constitute a challenge for proper computational detection and quantification of miRNAs. Further, miRNA detection often relies on the presence of a preferably complete genomic reference to align miRNA reads to a genomic locus or reference database, or extensive homology searches to exploit evolutionary conservation of a nearby species. Thus, two open questions in analyses of data from miRNA-seq are: (i) How can *isomiR* variation be detected and properly quantified and (ii) does a reference-free approach to miRNA detection exist and is it as sensitive as the conventional approach described above? Can it additionally be used to incorporate knowledge from other organisms?

To functionally characterise miRNAs, the delineation of miRNA target genes, i.e. those genes that could be silence by complimentary miRNA binding, is required and often accomplished using *in silico* target prediction tools (reviewed in @oulasPredictionMiRNATargets2015). Although prior knowledge from databases harbouring validated miRNA target interactions (MTIs) exists, target prediction and integration of other experimental data, e.g. from expression studies, remains challenging [@bayerLimiTTLinkMiRNAs2016].

Epigenetic regulation plays an important role in cell lineage development and disruption of such regulation might serve tumour initiation as well as tumour maintenance (see chapter \@ref(epigenetic)). However, mechanistic insight into how epigenetic lesions take effect (also in cooperation with or followed by ordinary genetic stimuli) is missing. Open questions with diagnostic and therapeutic implications include whether or not an initiating genetic hit (e.g. gain or loss of function) becomes secondary, once a downstream epigenetic lesion has occurred and altered the cellular state towards permanent tumourigenicity. Advances in microarray and next generation sequencing technology make it possible to assay different mechanisms of epigenetic regulation at high-resolution and in a large number of samples [@theencodeprojectconsortiumIdentificationAnalysisFunctional2007; @listerFindingFifthBase2009] but require specialised computational analysis (reviewed in @bockComputationalEpigenetics2008). Finally, DNA methylation as an important mechanism of epigenetic regulation is included, due to it's important contribution to cancer development and diagnosis [@kulisDNAMethylationCancer2010; @sekiIntegratedGeneticEpigenetic2015; @sunRelationshipDNAMethylation2019]. Profiling DNA methylation using microarray technology [@bibikovaHighDensityDNA2011] allows researchers to assay large number of samples across the whole genome. Computational analysis of data from such technologies aims to identify differentially methylated regions between two or more groups and search for functional enrichment in those regions [@lairdPrinciplesChallengesGenomewide2010; @bockAnalysingInterpretingDNA2012]. However, such approaches are complicated by the spatial interdependencies of individual CpG sites. Open questions include how data from single CpG sites can be aggregated into regions in order to capture higher-order methylation patterns across broader genomic regions. It is unclear whether such data can be analysed without the need of prior knowledge, e.g. the definition of genomic regions, location of CpG islands, promoters or other gene regulatory regions. Annotation of differentially methylated regions to nearby genes or known genomic features for further downstream interpretation of results is not straightforward and requires flexible and fast software solutions [@kondiliUROPAToolUniversal2017a].

### Objectives

Based on the previously described shortcomings of currently available methods for analysis of data obtained from high throughput experiments, two of the three main objectives of this thesis focus on methodological improvements in miRNA quantification and the analysis of DNA methylation pattern in arbitrary genomic regions. Another objective aims at the disclosure of cellular and molecular origins of ERMS, using multilayer datasets, applied bioinformatics and advanced methods from cell biology. Ultimately, methodological advancements will help to interpret and understand data from high-throughput experiments in Rhabdomyosarcoma and potentially lead to the disclosure of molecular pathways acting during tumour initiation and progression.

#### Reference-free and fine-grained analysis of data from miRNA-sequencing

The large number of miRNAs detected from previous high-throughput experiments across many species and experimental conditions represents a great data resource to guide detection and quantification of newly-created datasets. Sensitive homology searches against existing miRNA sequences might be a promising starting point for genome reference-free miRNA detection. A graph-based data structure with known miRNA sequences as nodes, and edges representing sequencing read(s) matching known miRNA sequences, might enable further fine-grained analyses: We postulate that closely connected components of the graph represent miRNA families across different *isomiR*s and species, and this allowing improved quantification on either family or *isomiR* levels by modulation of sensitivity of the homology search. Further, the sum of edge weights might be fed into differential expression analysis, readliy enabling streamlined downstream analysis.

#### Analysis of DNA methylation pattern in arbitrary genomic regions

Although a plethora of methods exist for data normalization and single-site CpG methylation analysis, a robust method that takes the spatial interdependencies of CpG sites into account and allows for unbiased genome-wide analysis is critically missing. Combination of differences of CpG site methylation estimated via one-sided two-sample Wilcoxon rank tests with a recently published method for grouping and correction of spatially correlated p-values [@pedersenCombpSoftwareCombining2012] might represent a novel and powerful statistical approach to detect differentially methylated regions. Importantly, this approach would eliminate the need for *a priori* knowledge of regulatory regions and allow for evaluation of arbitrary genomic regions. Integration of numerous existing normalization techniques and automated downstream gene set enrichment methods might additionally enhance usability and open the method for a wide variety of applications.

#### The cellular and molecular origin of embryonic rhabdomyosarcoma

Existing studies delineating the cellular and molecular origin of Rhabdomyosarcoma did not provide direct evidence for involvement of the myogenic lineage in tumour formation, but rather based their (valuable) findings on correlation with myogenic properties. The Cre/lox site-specific recombination system has been developed to create time- and tissue-specific mutations, e.g. to study effects of inducible gene knockouts [@feilInducibleCreMice2009]. We reasoned that employment of an inducible recombination system allowing permanent fluorescence labelling of muscle stem cell as well as their progeny might provide direct evidence of muscle stem cell tumourigenicity. Further, it would enable subsequent genomic and epigenomic analysis of purified tumour-propagating cells, as well as cancer subtype detection and investigation of clonal evolution in secondary recipients.
