### Knowledge gaps

The questions of how and why Rhabdomyosarcoma develops has already led to several studies set out to characterize the cellular origin and molecular alterations in RMS. In an extensive genomic analysis of tumors from 13 patients, Chen et. al.[@chenTargetingOxidativeStress2013] report an increased background mutation rate, increased number of SVs and CNVs in tumors from ERMS patients, compared to tumors from ARMS patients. However, no recurrent alteration was common to more than two patients and no hints for chromotripsis, as an explanation for large numbers of SVs, were observerd. Instead, they report large genomic heterogeneity at sites of known oncogenic drivers, like in Ras family genes or SHH/Wnt signalling. The somatic mutation theory introduced earlier (see chapter \@ref(smt)) puts forward the need for progressive accumulation of genome alterations that provide the cell with enabling abilities. Together with observations from Chen et. al., two open questions can be answered: RMS, and especially the ERMS subtype accumulated both, single nucleotide polymorphisms and larger copy number variations. Genomic alterations were not reoccuring among cancer samples, providing large cancer heterogeneity - a result that follows observations from large genome sequencing projects. Extending the somatic mutation theory, the genome theory explains cancer heterogeneity by introducing karyotype heterogeneity. Further, the genome theory assumes that initial genome context instability ignites a series of genome alterations from which *fittest* (in terms of cell proliferation and expansion) can be selected by an evolutionary process. An open question thus is which mechanisms induce genomic instability in RMS and how do accumulated genome alterations confer to fitness? Which hallmarks do they operate?

Following the stem cell theory of cancer (see chapter \@ref(csc)), the cancer cell of origin of embryonic rhabdomyosarcoma could possibly be either the muscle stem cell itself, or any of its potent progenitor cells, e.g. activated stem cells or pro-myogenic precursor cells. Elevated expression of Pax7 exclusively in fusion-negative RMS led (@tiffinPAX7ExpressionEmbryonal2003) to propose their origin to lie within the myogenic lineage. Transformation of precursor cell populations from the myogeneic lineage, with either expression of oncogeneic Kras [@hettmerSarcomasInducedDiscrete2011; @blumDistinctOverlappingSarcoma2013], or lineage-specific deletion of tumor supressor genes [@rubinEvidenceUnanticipatedRelationship2011], led to tumors that correlate with their *presumable* myogenic origin. However, it was not possible to discriminate the cellular tumor origin with transcriptional analysis of tumors expressing the Kras oncogenes [@@hettmerSarcomasInducedDiscrete2011]. Referenced studies are based on observed correlation with myogenic properties and lack direct experimental evidence, by leaving the possibility that other cell types could act as tumor initiator by e.g. cell migration mechanisms. Therefore, the cellular origin of RMS ultimately remains to be disclosed.

Bioinformatics

- Can miRNAs be detected without a genomic reference? Is a reference free detection as sensitive as a detection with genomic reference?
- Can miRNA quantification take into account isomirs? Can it be fed directly into differential expression analysis?
- miRNA target prediction 

- Can data from methylation arrays be aggregated into regions, to capture methylation across broader genomic regions? Can methylation data be analysed without prior knowledge, e.g. location of CpG islands, promoters etc.
- Annotation of differentially methylated regions for downstream analysis?

### Objectives

A host of applications exist in the field of RNA-Seq miRNA analyses, which attempt to quantify transcripts based on reference genomes or reference miRNAs, using different degrees of granularity including subsequences and precursors of miRNAs, deviating handling of technical and biological variation, offering a range of interfaces to the user (web, command line, graphical user interfaces), and bearing variable demands considering user expertise in informatics and biological interpretation.

