### Knowledge gaps

The search for the origin of Rhabdomyosarcoma has already led to several studies set out to characterize the cellular origin of, as well as molecular alterations occuring in RMS. Chen et. al performed extensive genomic analysis of tumors from 13 patients, but could not find recurrent alterations common to more than two patients. Instead, they report large genomic heterogeneity at sites of known oncogenic drivers, like the Ras gene family, that enabled cells to transform towards a cancer phenotype. The somatic mutation theory introduced earlier (see chapter \@ref(smt)) puts forward the need for progressive accumulation of genome alterations that provide the cell with enabling abilities. Together with observations from Chen et. al., two open questions can be answered: RMS, and especially the ERMS subtype accumulated both, single nucleotide polymorphisms and larger copy number variations. Genomic alterations were not reoccuring among cancer samples, providing large cancer heterogeneity - a result that follows observations from large genome sequencing projects. Extending the somatic mutation theory, the genome theory explains cancer heterogeneity by introducing karyotype heterogeneity. Further, the genome theory assumes that initial genome context instability ignites a series of genome alterations from which *fittest* (in terms of cell proliferation and expansion) can be selected by an evolutionary process. An open question thus is which mechanisms induce genomic instability in RMS and how do accumulated genome alterations confer to fitness? Which hallmarks do they operate?

Following the stem cell theory of cancer (see chapter \@ref(csc)), the cancer cell of origin of embryonic rhabdomyosarcoma could possibly be either the muscle stem cell itself, or any of its potent progenitor cells, e.g. activated stem cells or pro-myogenic precursor cells. Indeed, several studies suspect muscle stem cells to be the cellular origin of ERMS (Tiffin, Blum, Rubin), but others consider transdifferentiation from mesenchymal or endothelial progenitor cells possible (Hettmer, Drummond). 

### Objectives

A host of applications exist in the field of RNA-Seq miRNA analyses, which attempt to quantify transcripts based on reference genomes or reference miRNAs, using different degrees of granularity including subsequences and precursors of miRNAs, deviating handling of technical and biological variation, offering a range of interfaces to the user (web, command line, graphical user interfaces), and bearing variable demands considering user expertise in informatics and biological interpretation.

