Cancer has repeatedly been reported to be one of the leading causes of death for adults, adolescents and children around the world [@steliarova-foucherInternationalIncidenceChildhood2017], being responsible for nearly 10 million deaths in 2018. Cancer is however not a single disease entity, but rather a complex disease that can arise in multiple tissues, originating from multiple cell types and mechanisms. Currently, ~100 distinct [cancer types](https://www.cancer.gov/types) are known to emerge from interactions of hundreds to thousands of macromolecules. Although recent decades have witnessed some breakthroughs in molecular biology and cancer research in particular, and have generated unarguably valuable insights into variation in cancer initiation, progression, severity and treatment resistance, a clear vision and path for the cure of cancer is still missing [@nurseEndsUnderstanding1997;@koutsogiannouliComplexityCancerBiology2013]. 

The initial sequencing of the human genome [@internationalhumangenomesequencingconsortiumInitialSequencingAnalysis2001] has additionally marked the beginning of the (cancer) genomics era and founded modern biomedical research. The following decade was shaped by disruptive advances in DNA sequencing technology, revolutionizing not only cancer research [@macconaillExistingEmergingTechnologies2013], but also the way how genome-wide questions can be adressed. *Next generation sequencing* (NGS) technology enabled profiling of cancer genomes [@stephensComplexLandscapesSomatic2009;@pughMedulloblastomaExomeSequencing2012] and significantly enhanced the ability to study neoplastic transformation based on changes in the genome sequence. However, it introduced challenges in computational data analysis [@mardisDecadePerspectiveDNA2011; @wuCancerBioinformaticsNew2012] and transformation of the gained knowledge into an holistic understanding of cancer as a complex and dynamic system [@grizziCancerInitiationProgression2006; @sigstonEmergenceFrameworkCarcinogenesis2017].

The dynamic nature of cancer results in large intertumor heterogeneity and is one of the biggest challenges hampering successful patient treatment. Integration of patient data from multiple levels (e.g. genomics, epigenomics, clinical data, etc.) aims to distinguish patients with similar perceived cancer types but different requirements regarding successful treatment based on the cellular and molecular tumor profiles. Tailoring treatment to individual patients has been fittingly termed *personalized medicine* or *precision oncology* [@vincentComingEraPrecision2017;@senftPrecisionOncologyRoad2017] and requires integrative multi-omics approaches to identify therapeutic targets that are essential for tumor initiation or progression. To fulfill requirements of precision oncology and to support clinical decision making, appropriate computational methods for managing, integrating and analyzing large and complex data sets are needed [@singerBioinformaticsPrecisionOncology2017].

This thesis provides methodological development and application of software within the scope of computational cancer biology, focusing on the analysis of data from high-throughput experiments to support the characterization of the molecular basis and the investigation of mechanisms underlying cancer formation. The objectives are: to provide an algorithm for rapid detection and quantification of microRNAs, to develop a software pipeline for analysis and visualization of CpG-site methylation in a case-control setting, to investigate the cellular and molecular origin of a childhood cancer, embryonal rhabdomyosarcoma, by an integrative analysis of data generated from high-throughput experiments and to delineate mechanisms of tumor formation.
