Cancer remains to be one of the leading causes of death around the world, being responsible for nearly 10 million deaths in 2018 [@brayGlobalCancerStatistics2018]. Importantly, cancer is not a single disease entity, but rather a complex disease that can arise in multiple tissues, originating from different cell types and mechanisms. Currently, ~100 distinct [cancer types](https://www.cancer.gov/types) are known to emerge from interactions of hundreds to thousands of macromolecules. Recent decades of research have unarguably generated valuable insights into variations of cancer initiation, progression, severity and treatment resistance. However, a clear vision and path for the cure of cancer is still missing [@nurseEndsUnderstanding1997;@koutsogiannouliComplexityCancerBiology2013]. 

Efficiency of tumour treatment depends on and is affected by the cellular and molecular tumour profiles. However, individual tumours from different patients exhibit different molecular profiles and properties like cellular morphology, gene expression, metabolism, proliferation or metastatic potential. Such intratumoural heterogeneity is caused by cancer subtypes and hampers effective design of treatment strategies. It is one of the biggest challenges for successful cancer treatment. A promising approach to overcome intratumoural heterogeneity aims to identify individual patients with similar cancer subtypes and tailoring treatment to those patients [@vincentComingEraPrecision2017;@senftPrecisionOncologyRoad2017]. Termed *precision* medicine, it requires integration of patient data from multiple sources (e.g. genomics, epigenomics, clinical data, lifestyle and environment) to identify therapeutic targets that are essential for subtype-specific tumour initiation or progression. To fulfil those requirements and to support clinical decision making, appropriate computational methods for managing, integrating and analysing large and complex data sets are needed [@singerBioinformaticsPrecisionOncology2017].

The initial sequencing of the human genome [@internationalhumangenomesequencingconsortiumInitialSequencingAnalysis2001] has marked the beginning of the cancer genomics era and founded modern biomedical research. Disruptive advances in DNA sequencing technology revolutionised not only cancer research, but also the way how genome-wide questions can be addressed [@macconaillExistingEmergingTechnologies2013]. With the advent of *next generation sequencing* (NGS) technology, it has become possible to profile cancer genomes [@stephensComplexLandscapesSomatic2009;@pughMedulloblastomaExomeSequencing2012], which significantly enhanced the ability to study neoplastic transformation based on changes in the genome sequence. Notwithstanding, enourmous amounts of data generated by NGS introduced new challenges in computational data analysis [@mardisDecadePerspectiveDNA2011; @wuCancerBioinformaticsNew2012] and transformation of this data to gain a holistic understanding of the complex and dynamic systems of cancer remains challenging [@grizziCancerInitiationProgression2006; @sigstonEmergenceFrameworkCarcinogenesis2017].

This thesis provides methodological development and application of software within the scope of computational cancer biology, focusing on the analysis of data from high-throughput experiments to support the characterization of the molecular basis of cancer subtypes and the investigation of mechanisms underlying cancer formation. The objectives are: to provide an algorithm for rapid detection and quantification of small, regulatory RNA; to develop a software pipeline for analysis and visualization of CpG-site methylation in a case-control setting; to investigate the cellular and molecular origin of a childhood cancer, embryonal rhabdomyosarcoma, by an integrative analysis of data generated from high-throughput experiments and to delineate potential cancer subtypes and the mechanisms of tumour formation.
