This thesis provides methodological development in three areas of cancer bioinformatics, targeting detection and quantification of miRNAs, analysis of CpG-site methylation as an epigenetic mechanism and characterization of the cellular and molecular basis of Rhabdomyosarcoma. 

### MIRPIPE predicts and quantifies miRNA sequences with high sensitivity

Detected miRNA sequences from MIRPIPE were validated with two complimentary approaches based on genomic mapping and found to be as sensitive as existing methods, recovering 84% and 96% of reference miRNAs respectively. Quantification results of MIRPIPE effectively reproduced quantification of two gold-standard datasets with Spearman rank correlation values of 0.68 and 0.69, respectively. Surprisingly, specificity of MIRPIPE was found to be higher compared to the approach based on genomic mapping, well grounded in MIRPIPEs strategy to filter out lowly abundant reads prior to graph-based analysis. Functional characterization of detected miRNA sequences by delineation of miRNA target genes was adressed in a follow-up project termed LimiTT [@bayerLimiTTLinkMiRNAs2016] with contributions from the author of this thesis. Briefly, LimiTT integrates several databases of experimentally validated miRNA-target interactions (MTIs) and additionally allows to utilize data from RNA expression experiments to weight important MTIs via built-in MTI set enrichment analyis.

The employment of a graph-based data structure for results from homology searches against a database of known miRNAs is a novel and unique approach that allows handling of miRNA sequences on both, the miRNA family and *isoMiR* level and represents a feature that was previously missing. Interestingly, the approach resembles current methods of transcript-level quantification in analysis of RNA-sequencing data termed *pseudoalignment* [@brayNearoptimalProbabilisticRNAseq2016; @patroSalmonProvidesFast2017]. Pseudoalignment does not require mapping to a genomic reference, but performs probabilistic assignment of sequencing reads to known transcripts, producing a list of compatible transcripts per sequencing read using matching of *k-mer* contents. Aggregation of so-called transcript-compatibility counts results in gene-level quantification, similar to MIRPIPEs summation of *isoMiR* counts to produce miRNA family level counts. As miRNA reference databases might grow in the near future, adoption of *pseudoalignment* for miRNA quantification seems to a good replacement of the time-consuming homology searches and promises to speed-up runtime in several orders of magnitude.

### ADMIRE 

Most of the existing methods for analysis of CpG methylation data feature detection of differential methylation at individual CpG sites only, or, if at all, are limited to pre-defined genomic regions, like CpG islands or gene regulatory promotors. In contrast, the unique statistical approach implemented in ADMIRE permits methlyation data from CpG sites to be combined into arbitrary genomic regions, while taking into account their spatial correlation. The approach has been shown to gain sensitivity when dealing with small sample numbers or when DNA methylation is changed globally, e.g. as discussed for the CpG island methylator phenotype (see chapter \ref(@epigenetic)). Two datasets were used to assess sensitivity and significance of results obtained from ADMIRE: Investigation of DNA methylation change in a study of permanent atrial fibrillation (AF) showed high sensitivity of ADMIRE, which found 20 regions to be differentially methylated, although only 11 samples have been used as input. Its direct competitor, RnBeads, reported one region with higher methylation in AF, which was not reported by ADMIRE. Further, ADMIRE detected 14 additional regions up to 10kB and subsequent gene set enrichment analysis confirmed results of previously conducted GWAS studies. A second dataset was used to show ADMIREs performance in large sample cohorts by analyzing 689 samples from a study analyzing DNA methylation as an intermediary of genetic risk in rheumatic arthritis (RA). Confirming implication of the MHC region from the original study, ADMIRE additionally liked differential methylation in the T-cell activation and T-cell receptor signaling pathway to RA, proving its scaleability and applicability in large clinical studies.

### Lineage-tracing of p53-deficient muscle stem cells reveals cellular origin of ERMS

Employment of the inducible Cre/lox site-specific recombination system enabled creation of mice with muscle stem cell-specific expression of the Cre recombinase (Pax7^CreERT2^). Those mice further carried two lox-p sites in the Trp53 gene, as well as a Rosa26::lsl Tomato allele, enabling muscle stem cell specific inactivation of the tumor supressor p53 (SC^p53^) and permanent flourescent lineage tracing of p53-deficient muscle stem cells by Tomato expression upon treatment with Tamoxifen (TAM). Mdx mice harbouring the inducible system exhibited tumor formation at sites of musculature extremities or the trunk upon TAM administration. Lineage-traced tumors were histopathologically classified as embryonic Rhabdomyosarcoma and immunopositive for Desmin, MyoD and MyoG, clearly indicating their origin from the muscular lineage. TAM-treated wildtype or mdx mice never developed tumors and TAM-treated SC^p53^ mice only developed tumors upon consecutive bouts of Cardiotoxin-induced injury of the Tibalis anterior muscle, showing that muscle stem cell-specific loss of p53 in a regenerative environment is sufficient to generate RMS, or conversely, that a regenerative environment enables RMS formation upon muscle stem cell-specific inactivation/loss of p53. Lineage-tracing additionally enabled separation and purification of muscle stem cells into non-lineage-traced and lineage-traced tumor propagating cells (TPCs) using fluorescence-activated cell sorting (FACS). Importantly, transplantation of lineage-traced p53 deficient muscle stem cells into immunocompromised mdx-nude mice generated tumors already two weeks after injection. These data confirmed the hypothesis put forward by the stem cell theory introduced earlier (see chapter \ref(@csc)) and disclosed the oncogeneic capacity of muscle stem cells as cellular origin of embryonal RMS. 

### Distinct copy number variation constitute molecular origin of RMS

Whole-exome DNA sequencing of purified TPCs and matched normal samples was followd by subsequent genome analysis to identify tumor-promoting mutations. In 20 out of 21 specimen, discrete and dramatic copy number amplifications were found to be prevailing mutations and positional mapping of those revealed defined chromosomal regions harbouring known mutational targets in ERMS, including Yap1 (Tremblay et al., 2014; 8/22; 36%), C-met (Fleischmann et al., 2003; Taulli et al., 2006; 5/22; 23%), Jun (Durbin et al., 2009; 1/22; 4.5%), and Cdk4/Gli1/Os9 (Liu et al., 2014; 1/22; 4.5%). Other mutational targets so far not implicated in ERMS formation but in other types of cancer included Rras (Flex et al., 2014), Kdm4d (Soini et al., 2015), Bap1 (Robertson et al., 2017), Mcm4 (Polotskaia et al., 2015; Shima et al., 2007), and Eloc (Sato et al., 2013). Interestingly, TPCs did not exhibit accumulated somatic single-nucleotide variation, indicating that EMRS does not follow the classic mechanism proposed by the somatic mutation theory (see chapter \ref(@smt)), but concordant with recent observations that soft-tissue sarcomas are predominantly characterized by copy number changes. 

It is widely accepted that overexpression of oncogenes or loss of tumor supressor genes can be a crucial molecular event resulting in tumor initiation, but a question still under debate is whether maintenance of tumorigeneicity depends as well on the transforming molecular event. Indeed, the phenomenon of *oncogene addiction* (i.e. the physiological dependence of cancer cells on oncogenes, (@weinsteinCANCEREnhancedAddiction2002)) has been described for several cancer types and offers possibilities for therapeutic intervention by specific drugs targeting oncogene expression. For example, targeting Yap1 expressing TPCs with short hairpin RNA (shRNA)-mediated knockdown of Yap1 resulted in cell death, indicating the dependence of TPCs on distinct regulatory networks facilitated by Yap1 expression. However, such intervention requires personalized therapeutic approaches often not yet implemented in clinical settings. Additionally, the cancer phenotype might not be reversed by blocking expression of an oncogene, if oncogene-mediated genome instability induced subsequent mutations that enable cells to escape oncogene dependence.

### Duxbl

- KDM upregulation upon Duxbl overexpression -> permissive chromatin
- Duxbl quick downregulation
- tissue field organization theory true for secondary recipient
- epigenetic hit and run vs. oncogene addiction

Conclusion:
Moreover, these data support the notion that maintenance of stem cell quiescence provides a cellular mechanism to suppress tumorigenesis and are consistent with previous reports demonstrating that muscle injury is required to elicit RMS formation. 
- reintroduction of p53? 
