This thesis provides methodological development in three areas of cancer bioinformatics, targeting detection and quantification of miRNAs, analysis of CpG-site methylation as an epigenetic mechanism and characterization of the cellular and molecular basis of cancer. 

### MIRPIPE predicts and quantifies miRNA sequences with high sensitivity

Detected miRNA sequences from MIRPIPE were validated with two complimentary approaches based on genomic mapping and found to be as sensitive as existing methods, recovering 84% and 96% of reference miRNAs respectively. Quantification results of MIRPIPE effectively reproduced quantification of two gold-standard datasets with Spearman rank correlation values of 0.68 and 0.69, respectively. Surprisingly, specificity of MIRPIPE was found to be higher compared to the approach based on genomic mapping, well grounded in MIRPIPEs strategy to filter out lowly abundant reads prior to graph-based analysis. Functional characterization of detected miRNA sequences by delineation of miRNA target genes was adressed in a follow-up project termed LimiTT [@bayerLimiTTLinkMiRNAs2016] with contributions from the author of this thesis. Briefly, LimiTT integrates several databases of experimentally validated miRNA-target interactions (MTIs) and additionally allows to utilize data from RNA expression experiments to weight important MTIs via built-in MTI set enrichment analyis.

The employment of a graph-based data structure for results from homology searches against a database of known miRNAs is a novel and unique approach that allows handling of miRNA sequences on both, the miRNA family and *isoMiR* level and represents a feature that was previously missing. Interestingly, the approach resembles current methods of transcript-level quantification in analysis of RNA-sequencing data termed *pseudoalignment* [@brayNearoptimalProbabilisticRNAseq2016; @patroSalmonProvidesFast2017]. Pseudoalignment does not require mapping to a genomic reference, but performs probabilistic assignment of sequencing reads to known transcripts, producing a list of compatible transcripts per sequencing read using matching of *k-mer* contents. Aggregation of so-called transcript-compatibility counts results in gene-level quantification, similar to MIRPIPEs summation of *isoMiR* counts to produce miRNA family level counts. As miRNA reference databases might grow in the near future, adoption of *pseudoalignment* for miRNA quantification seems to a good replacement of the time-consuming homology searches and promises to speed-up runtime in several orders of magnitude.

### ADMIRE 

Most of the existing methods for analysis of CpG methylation data feature detection of differential methylation at individual CpG sites only, or, if at all, are limited to pre-defined genomic regions, like CpG islands or gene regulatory promotors. In contrast, the unique statistical approach implemented in ADMIRE permits methlyation data from CpG sites to be combined into arbitrary genomic regions, while taking into account their spatial correlation. The approach has been shown to gain sensitivity when dealing with small sample numbers or when DNA methylation is changed globally, e.g. as discussed for the CpG island methylator phenotype (see chapter \ref(@epigenetic)). Two datasets were used to assess sensitivity and significance of results obtained from ADMIRE: Investigation of DNA methylation change in a study of permanent atrial fibrillation (AF) showed high sensitivity of ADMIRE, which found 20 regions to be differentially methylated, although only 11 samples have been used as input. Its direct competitor, RnBeads, reported one region with higher methylation in AF, which was not reported by ADMIRE. Further, ADMIRE detected 14 additional regions up to 10kB and subsequent gene set enrichment analysis confirmed results of previously conducted GWAS studies. A second dataset was used to show ADMIREs performance in large sample cohorts by analyzing 689 samples from a study analyzing DNA methylation as an intermediary of genetic risk in rheumatic arthritis (RA). Confirming implication of the MHC region from the original study, ADMIRE additionally liked differential methylation in the T-cell activation and T-cell receptor signaling pathway to RA, proving its scaleability and applicability in large clinical studies.

### Lineage-tracing in p53-deficient muscle stem cells reveals cellular origin of RMS

### Distinct copy number variation constitute molecular origin of RMS

### Duxbl

- KDM upregulation upon Duxbl overexpression -> permissive chromatin
- Duxbl quick downregulation
- epigenetic hit and run vs. oncogene addiction
