This thesis provides methodological development in three areas of cancer bioinformatics, targeting detection and quantification of miRNAs, analysis of CpG-site methylation as an epigenetic mechanism and characterization of the cellular and molecular basis of Rhabdomyosarcoma. 

### MIRPIPE predicts and quantifies miRNA sequences with high sensitivity

Detected miRNA sequences from MIRPIPE were validated with two complimentary approaches based on genomic mapping and found to be as sensitive as existing methods, recovering 84% and 96% of reference miRNAs respectively. Quantification results of MIRPIPE effectively reproduced quantification of two gold-standard datasets with Spearman rank correlation values of 0.68 and 0.69, respectively. Surprisingly, specificity of MIRPIPE was found to be higher compared to the approach based on genomic mapping, well grounded in MIRPIPEs strategy to filter out lowly abundant reads prior to graph-based analysis. Functional characterization of detected miRNA sequences by delineation of miRNA target genes was adressed in a follow-up project termed LimiTT [@bayerLimiTTLinkMiRNAs2016] with contributions from the author of this thesis. Briefly, LimiTT integrates several databases of experimentally validated miRNA-target interactions (MTIs) and additionally allows to utilize data from RNA expression experiments to weight important MTIs via built-in MTI set enrichment analyis.

The employment of a graph-based data structure for results from homology searches against a database of known miRNAs is a novel and unique approach that allows handling of miRNA sequences on both, the miRNA family and *isoMiR* level and represents a feature that was previously missing. Interestingly, the approach resembles current methods of transcript-level quantification in analysis of RNA-sequencing data termed *pseudoalignment* [@brayNearoptimalProbabilisticRNAseq2016; @patroSalmonProvidesFast2017]. Pseudoalignment does not require mapping to a genomic reference, but performs probabilistic assignment of sequencing reads to known transcripts, producing a list of compatible transcripts per sequencing read using matching of *k-mer* contents. Aggregation of so-called transcript-compatibility counts results in gene-level quantification, similar to MIRPIPEs summation of *isoMiR* counts to produce miRNA family level counts. As miRNA reference databases might grow in the near future, adoption of *pseudoalignment* for miRNA quantification seems to a good replacement of the time-consuming homology searches and promises to speed-up runtime in several orders of magnitude.

### Analysis of DNA methylation in arbitrary genomic regions with ADMIRE

Most of the existing methods for analysis of CpG methylation data feature detection of differential methylation at individual CpG sites only, or, if at all, are limited to pre-defined genomic regions, like CpG islands or gene regulatory promotors. In contrast, the unique statistical approach implemented in ADMIRE permits methlyation data from CpG sites to be combined into arbitrary genomic regions, while taking into account their spatial correlation. The approach has been shown to gain sensitivity when dealing with small sample numbers or when DNA methylation is changed globally, e.g. as discussed for the CpG island methylator phenotype (see chapter \@ref(epigenetic)). Two datasets were used to assess sensitivity and significance of results obtained from ADMIRE: Investigation of DNA methylation change in a study of permanent atrial fibrillation (AF) showed high sensitivity of ADMIRE, which found 20 regions to be differentially methylated, although only 11 samples have been used as input. Its direct competitor, RnBeads, reported one region with higher methylation in AF, which was not reported by ADMIRE. Further, ADMIRE detected 14 additional regions up to 10kB and subsequent gene set enrichment analysis confirmed results of previously conducted GWAS studies. A second dataset was used to show ADMIREs performance in large sample cohorts by analyzing 689 samples from a study analyzing DNA methylation as an intermediary of genetic risk in rheumatic arthritis (RA). Confirming implication of the MHC region from the original study, ADMIRE additionally liked differential methylation in the T-cell activation and T-cell receptor signaling pathway to RA, proving its scaleability and applicability in large clinical studies.

### Lineage-tracing of p53-deficient muscle stem cells reveals cellular origin of ERMS

Employment of the inducible Cre/lox site-specific recombination system enabled creation of mice with muscle stem cell-specific expression of the Cre recombinase (Pax7^CreERT2^). Those mice further carried two lox-p sites in the Trp53 gene, as well as a Rosa26::lsl Tomato allele, enabling muscle stem cell specific inactivation of the tumor supressor p53 (SC^p53^) and permanent flourescent lineage tracing of p53-deficient muscle stem cells by Tomato expression upon treatment with Tamoxifen (TAM). Mdx mice harbouring the inducible system exhibited tumor formation at sites of musculature extremities or the trunk upon TAM administration. Lineage-traced tumors were histopathologically classified as embryonic Rhabdomyosarcoma and immunopositive for Desmin, MyoD and MyoG, clearly indicating their origin from the muscular lineage. TAM-treated wildtype or mdx mice never developed tumors and TAM-treated SC^p53^ mice only developed tumors upon consecutive bouts of Cardiotoxin-induced injury of the Tibalis anterior muscle, showing that muscle stem cell-specific loss of p53 in a regenerative environment is sufficient to generate RMS, or conversely, that a regenerative environment enables RMS formation upon muscle stem cell-specific inactivation/loss of p53. Lineage-tracing additionally enabled separation and purification of muscle stem cells into non-lineage-traced and lineage-traced tumor propagating cells (TPCs) using fluorescence-activated cell sorting (FACS). Importantly, transplantation of lineage-traced p53 deficient muscle stem cells into immunocompromised mdx-nude mice generated tumors already two weeks after injection. These data confirmed the hypothesis put forward by the stem cell theory introduced earlier (see chapter \@ref(csc)) and disclosed the oncogeneic capacity of muscle stem cells as cellular origin of embryonal RMS. 

### Distinct copy number variation constitute molecular origin of RMS

Whole-exome DNA sequencing of purified TPCs and matched normal samples was followd by subsequent genome analysis to identify tumor-promoting mutations. In 20 out of 21 specimen, discrete and dramatic copy number amplifications were found to be prevailing mutations and positional mapping of those revealed defined chromosomal regions harbouring known mutational targets in ERMS, including Yap1 [@tremblayHippoTransducerYAP12014], C-met [@taulliValidationMetTherapeutic2006], Jun [@durbinJNK1DeterminesOncogenic2009], and Cdk4/Gli1/Os9 [@liuChromosomalGeneticImbalances2014]. Interestingly, TPCs did not exhibit accumulated somatic single-nucleotide variation, indicating that EMRS does not follow the classic mechanism proposed by the somatic mutation theory (see chapter \@ref(smt)), but concordant with recent observations that soft-tissue sarcomas are predominantly characterized by copy number changes. 

It is widely accepted that overexpression of oncogenes or loss of tumor supressor genes can be a crucial molecular event resulting in tumor initiation, but a question still under debate is whether maintenance of tumorigeneicity depends as well on the transforming molecular event. Indeed, the phenomenon of *oncogene addiction* (i.e. the physiological dependence of cancer cells on oncogenes, (@weinsteinCANCEREnhancedAddiction2002)) has been described for several cancer types and offers possibilities for therapeutic intervention by specific drugs targeting oncogene expression. For example, targeting Yap1 expressing TPCs with short hairpin RNA (shRNA)-mediated knockdown of Yap1 resulted in cell death, indicating the dependence of TPCs on distinct regulatory networks facilitated by Yap1 expression. However, such intervention requires personalized therapeutic approaches often not yet implemented in clinical settings. Additionally, the cancer phenotype might not be reversed by blocking expression of an oncogene, if oncogene-mediated genome instability induced subsequent mutations that enable cells to escape oncogene dependence.

### Amplification of zygotic Dux factors defines a molecular subtype of cancer

Several mice displayed amplification of a poorly described locus without any known oncogene on chromosome 14qA3. Analysis of genomic synteny (i.e. the pyhsical co-localization of genetic loci) between species revealed Duxbl, a member of the Dux family of homeobox-containing transcription factors, to be located in synteny with human DuxB. Interestingly, DuxB and its paralog, DuxA, were recently shown to be expressed exclusively at the totipotent 8-cell stage in early zygotes [@madissoonCharacterizationTargetGenes2016a]. Furthermore, the founding member of the Dux transcription factor family, Dux4, and its murine homolg Dux, were shown to be responsible for driving cleavage-stage gene expression known as zygotic gene activation (ZGA) in totipotent embryonic stem cells [@leidenrothFamilyHistoryDUX42010; @hendricksonConservedRolesMouse2017a; @whiddonConservationInnovationDUX4family2017]. Those findings led to the speculation that Dux transcription factors might act at a putative interface of stem cell potency and tumor formation, but the discovery of their role was hampered until recent technological advancements allowed experimental assessment of very little input material, as in the case of early zygotes.

To test whether Dux driven activation of zygotic genes plays a role in human ERMS, but was previously missed due to poor genomic annotation, an integrative analysis was set out to translate findings from whole-exome sequencing in mouse to transcriptome sequencing in human cancer patients. Intriguingly, 54 tumors (~10%) that were positive for expression of Dux4, DuxA or DuxB and showed a cleavage-stage-specific expression signature were detected within a previously published discovery cohorts of human ERMS patients [@davicioniMolecularClassificationRhabdomyosarcoma2009;@williamsonFusionGeneNegative2010;@chenTargetingOxidativeStress2013]. To test whether increased expression of Dux genes is restricted to ERMS or can also be associated with other malignancies, expression data from ~10,000 cancer patients from The Cancer Genome Atlas [@hoadleyMultiplatformAnalysis122014] was integrated as well. Additional 349 patients displayed distinct expression of Dux family members either in combination or alone. Cancer onset and type in these patients was highly variable and suggests that Dux transcription factors define a molecular subtype of a broad range of human cancers, including ERMS.

The selection of suitable targets for individual tumor therapeutics critically depends on molecular insights into mechanisms of tumor initiation and progression. To gain understanding on the action of Dux transcription factors, Duxbl was overexpressed in wild-type muscle stem cells and resulted in the emergence of immortalized and morphologically rounded clones prone to spontaneuosly form epithelial-like spherical aggregates. Subcutaneous transplantation of clones formed tumors at the site of engraftment and clearly demonstrated that overexpression of Duxbl is sufficient for neoplastic transformation of muscle stem cells *in vivo*. Interestingly, transformed cells contributed to myofiber formation when injected directly into the strong pro-differentiation environment of the *tibialis anterior* muscle, further supporting the supressive role of functional tissue in cancer progression [@bissellPuttingTumoursContext2001; chapter \@ref(development)]. Expression analysis of isolated clones revealed upregulation of the histone lysine demethylase Kdm4d but no expression of myogenic determinants like Myf5, MyoD and MyoG, hinting towards a *lineage independent* mechanism of cell transformation. Instead, dramatic induction of genes involved in epithelial cell proliferation and encoding for integrins, collagens, cadherins and proto-cadherins was observed, along with expression of pluripotency factors Sox2 and Klf4, which are instrumental to facilitate an essential mesenchymal-to-epithelial transition during reprogramming of somatic cells to induced pluripotent stem cells (iPSC) [@liMesenchymaltoEpithelialTransitionInitiates2010a]. Taken together, the overexpression of Duxbl confers cellular plasticity through Kdm4-mediated permissive chromatin [@labbeHistoneLysineDemethylase2013], which allows for induction of a MET-like transition that initiates plastic tumorigenic colonies. Most likely, the establishment of truly metastatic niches for tumor outgrowth requires a secondary event but does not depent on sustained expression of Duxbl. However, subsequent therapeutic intervention of tumorigenicity conferred by DuxB/Duxbl via an epigenetic *hit-and-run* [@saundersonHitandrunEpigeneticEditing2017] event could not make use of strategies for classical *oncogene addiction*, but requires novel therapeutic approaches.
