This thesis provides methodological development in three areas of cancer bioinformatics, targeting detection and quantification of miRNAs, analysis of CpG-site methylation affecting epigenetic mechanism and characterization of the cellular and molecular basis of Rhabdomyosarcoma including detection of cancer subtypes. 

### Sensitive computational quantification of miRNA sequences from high-throughput sequencing

MiRNA sequences from MIRPIPE were validated with two complimentary approaches based on genomic mapping and found to be as sensitive as existing methods, recovering 84% and 96% of reference miRNAs respectively. Quantification results of MIRPIPE effectively recapitulated quantification of two gold-standard datasets with Spearman rank correlation values of 0.68 and 0.69, respectively. Specificity of MIRPIPE was higher compared to the approach based on genomic mapping, based on MIRPIPEs strategy to filter out lowly abundant reads prior to graph-based analysis. Characterization of detected miRNA sequences by delineation of putative miRNA target genes was pursued in a follow-up project termed LimiTT [@bayerLimiTTLinkMiRNAs2016] with contributions from the author of this thesis. Briefly, LimiTT integrates several databases of experimentally validated miRNA-target interactions (MTIs) and additionally allows utilisation of data from RNA expression experiments to weight important MTIs via built-in MTI set enrichment analysis.

The employment of a graph-based data structure for results from homology searches against a database of known miRNAs is a novel and unique approach allowing handling of miRNA sequences miRNA family and *isoMiR* levels, a feature that was previously missing. The approach is similar to current methods of transcript-level quantification in analysis of RNA-sequencing data termed *pseudoalignment* [@brayNearoptimalProbabilisticRNAseq2016; @patroSalmonProvidesFast2017]. Pseudoalignment does not require mapping to a genomic reference, but performs probabilistic assignment of sequencing reads to known transcripts, producing a list of compatible transcripts per sequencing read using matching of *k-mer* contents. Aggregation of so-called transcript-compatibility counts results in gene-level quantification, similar to MIRPIPEs summation of *isoMiR* counts to produce miRNA family level counts. Since miRNA reference databases might grow in the near future, adoption of *pseudoalignment* for miRNA quantification seems to be a good replacement for time-consuming homology searches and promises to speed-up runtime by several orders of magnitude.

### Computational analysis of DNA methylation in arbitrary genomic regions

Most of the existing methods for analysis of CpG methylation data only feature detection of differential methylation at individual CpG sites. Thus, such approaces are limited to pre-defined genomic regions, such as CpG islands or gene regulatory promotors. In contrast, the unique statistical approach implemented in ADMIRE permits combination of methylation data from CpG sites with arbitrary genomic regions, while considering their spatial correlation. The approach has been shown to gain sensitivity when dealing with small sample numbers or when DNA methylation is changed globally, e.g. as discussed for the CpG island methylator phenotype (see chapter \@ref(epigenetic)). Two datasets were used to assess sensitivity and significance of results obtained from ADMIRE: Investigation of DNA methylation changes in a study of permanent atrial fibrillation (AF) showed high sensitivity of ADMIRE, which identified 20 regions differentially methylated, although only 11 samples were used as input. Its direct competitor, RnBeads, reported one region with higher methylation in AF, which was not reported by ADMIRE. Furthermore, ADMIRE detected 14 additional regions up to 10 kB and subsequent gene set enrichment analysis confirmed results of previously conducted GWAS studies. A second dataset was used to analyse ADMIRE's performance in large sample cohorts. 689 samples from a study analysing DNA methylation as an intermediary of genetic risk in rheumatic arthritis (RA) were analysed. In addition ADMIRE detected differential methylation in the T-cell activation and T-cell receptor signalling pathway to RA, thereby confirming implication of the MHC region from the original study and proving its scalability and applicability in large clinical studies.

### Lineage-tracing reveals cellular origin of ERMS and enables in-depth analysis of cancer stem cells

Mice expressing the Cre recombinase (Pax7^CreERT2^) in muscle stem cells were crossed to a strain carrying two lox-p sites in the Trp53 gene and the Rosa26::lsl Tomato allele, thereby enabling muscle stem cell specific inactivation of the tumour suppressor p53 (SC^p53^) and permanent fluorescent lineage tracing of p53-deficient muscle stem cells by Tomato expression upon treatment with Tamoxifen (TAM). Mdx mice harbouring the inducible system exhibited tumour formation at sites of musculature extremities or the trunk after TAM administration. Lineage-traced tumours were histopathologically classified as embryonic Rhabdomyosarcoma and were stained positive for myogenic factors, clearly indicating their origin from the muscular lineage. TAM-treated wildtype or mdx mice never developed tumours and TAM-treated SC^p53^ mice only developed tumours upon consecutive bouts of Cardiotoxin-induced injury of the Tibalis anterior muscle, demonstrating that muscle stem cell-specific loss of p53 in a regenerative environment is sufficient to generate RMS. Lineage-tracing enabled separation and purification of RMS cells into non-lineage-traced and lineage-traced tumour propagating cells (TPCs) using fluorescence-activated cell sorting (FACS). Importantly, transplantation of lineage-traced p53 deficient muscle stem cells into immunocompromised mdx-nude mice generated tumours already two weeks after injection. These data confirmed the hypothesis put forward by the stem cell theory introduced earlier (see chapter \@ref(csc)) and disclosed the cellular origin of embryonal RMS in the p53^-/-^/mdx model. 

### Computational analysis of copy number variation reveals molecular origin of ERMS

Whole-exome DNA sequencing of purified TPCs and matched normal samples was followed by subsequent genome analysis to identify tumour-associated mutations. In 20 out of 21 specimen, discrete and dramatic copy number amplifications were identified as the prevailing mutations. Positional mapping revealed defined chromosomal regions harbouring known mutational targets in ERMS, including Yap1 [@tremblayHippoTransducerYAP12014], C-met [@taulliValidationMetTherapeutic2006], Jun [@durbinJNK1DeterminesOncogenic2009], and Cdk4/Gli1/Os9 [@liuChromosomalGeneticImbalances2014]. Interestingly, TPCs did not accumulate somatic single-nucleotide variations, indicating that EMRS does not follow the classic mechanism proposed by the somatic mutation theory (see chapter \@ref(smt)) but are predominantly characterised by copy number changes. 

It is widely accepted that overexpression of oncogenes or loss of tumour suppressor genes is a crucial molecular event resulting in tumour initiation, but it is unclear whether maintenance of tumourigenicity depends on the transforming molecular event as well. The phenomenon of *oncogene addiction* (i.e. the physiological dependence of cancer cells on oncogenes, (@weinsteinCANCEREnhancedAddiction2002)) has been described for several cancer types and offers opportunities for therapeutic intervention by targeting oncogene expression with specific drugs. In fact, knockdown of Yap1 in Yap1-expressing TPCs using short hairpin RNA (shRNA) resulted in cell death, indicating the dependence of TPCs on distinct regulatory networks facilitated by Yap1 expression. However, such intervention requires personalized therapeutic approaches often not yet implemented in clinical settings. Additionally, the cancer phenotype might not be reversed by blocking expression of an oncogene, if oncogene-mediated genome instability induced subsequent mutations enable cells to escape oncogene dependence.

### Integrative analysis of zygotic Dux factors defines a new cancer subtype

Several mice displayed amplification of a poorly described locus without any known oncogene on chromosome 14qA3. Analysis of genomic synteny (i.e. the physical co-localization of genetic loci) between different species revealed that Duxbl is located in synteny with human DuxB, a member of the Dux family of homeobox-containing transcription factors. Interestingly, DuxB and its paralog, DuxA, were recently shown to be expressed exclusively at the totipotent 8-cell stage in early zygotes [@madissoonCharacterizationTargetGenes2016a]. Furthermore, the founding member of the Dux transcription factor family, Dux4, and its murine homolog Dux, are responsible for driving cleavage-stage gene expression known as zygotic gene activation (ZGA) in totipotent embryonic stem cells [@leidenrothFamilyHistoryDUX42010; @hendricksonConservedRolesMouse2017a; @whiddonConservationInnovationDUX4family2017]. Those findings led to the speculation that Dux transcription factors might act at a putative interface of stem cell potency and tumour formation. So far, a more detailed analysis was difficult until recent technological advancements allowed experimental assessment of very little input material, as in the case of early zygotes.

To test whether Dux-driven activation of zygotic genes plays a role in human ERMS, an integrative analysis was designed to translate findings from whole-exome sequencing in mouse to transcriptome sequencing in human cancer patients. Intriguingly, 54 tumours (~10%), which expressed Dux4, DuxA or DuxB, showed a cleavage-stage-specific expression signature in a previously published discovery cohorts of human ERMS patients [@davicioniMolecularClassificationRhabdomyosarcoma2009;@williamsonFusionGeneNegative2010;@chenTargetingOxidativeStress2013]. To test whether increased expression of Dux genes is restricted to ERMS or is associated with other malignancies, expression data from ~10,000 cancer patients from The Cancer Genome Atlas [@hoadleyMultiplatformAnalysis122014] was used for further molecular analysis. Interestingly, 349 patients displayed distinct expression of Dux family members either in combination or alone. The onset of cancer and the type of cancer was highly variable in these patients, suggesting that Dux transcription factors define a molecular subtype of a broad range of human cancers, including ERMS.

### Gene expression analysis reveals epigenetic plasticity conferred by tumourigenic Duxbl

The selection of suitable targets for individual tumour therapeutics critically depends on molecular insight into mechanisms of tumour initiation and progression. To gain understanding on the action of Dux transcription factors, Duxbl was overexpressed in wild-type muscle stem cells, which resulted in the emergence of immortalised and morphologically rounded clones prone to spontaneously form epithelial-like spherical aggregates. Subcutaneous transplantation of clones formed tumours at the site of engraftment clearly demonstrating that overexpression of Duxbl is sufficient for neoplastic transformation of muscle stem cells *in vivo*. Interestingly, transformed cells contributed to myofiber formation when injected directly into the strong pro-differentiation environment of the *tibialis anterior* muscle, further supporting the suppressive role of functional tissue in cancer progression (@bissellPuttingTumoursContext2001, chapter \@ref(development)). Expression analysis of isolated clones revealed upregulation of the histone lysine demethylase Kdm4d but no expression of myogenic determinants like Myf5, MyoD and MyoG, suggesting a *lineage independent* mechanism of cell transformation. Instead, dramatic induction of genes involved in epithelial cell proliferation and coding for integrins, collagens, cadherins and proto-cadherins was observed, along with expression of pluripotency factors Sox2 and Klf4. These genes are instrumental to facilitate mesenchymal-to-epithelial transition during reprogramming of somatic cells to induce pluripotent stem cells [@liMesenchymaltoEpithelialTransitionInitiates2010a]. Taken together, the overexpression of Duxbl confers cellular plasticity through Kdm4-mediated permissive chromatin [@labbeHistoneLysineDemethylase2013], which allows induction of a MET-like transition that initiates growth of tumourigenic colonies. Most likely, the establishment of truly metastatic niches for tumour outgrowth requires a secondary oncogenic event but might not depend on sustained expression of Duxbl. In such a scenario, therapeutic supression of tumourigenicity conferred by DuxB/Duxbl via an epigenetic *hit-and-run* [@saundersonHitandrunEpigeneticEditing2017] event, would be unable to take advantage of classical *oncogene addiction*, but would require novel therapeutic ideas.
